incubator/hnc/test/conversion/conversion_test.go

```diff
@@ -59,7 +59,7 @@ var _ = Describe("Conversion from v1alpha1 to v1alpha2", func() {
 		TearDownHNC("")
 	})
 
-	It("should convert subnamespace anchors", func() {
+	It("should convert subnamespace anchors and subnamespace-of annotation", func() {
```

Comment:
Maybe include a test case where _both_ are defined?

Reply:
Added and found that we will need to decide which one takes precedence if both exists in the conversion. PTAL at the new test and the reconciler behavior.